Sources/DistributedSystem/DistributedSystem.swift

```diff
@@ -178,6 +178,7 @@ public class DistributedSystem: DistributedActorSystem, @unchecked Sendable {
 
     private var discoveryManager: DiscoveryManager
     private var syncCallManager: SyncCallManager
+    private let compression: UInt8
```

Comment:
Shouldn't it be a `enum Compressor` with `LZ4 == 0`.

Reply:
Let's do it if will decide to use more than one compression.